app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,24 @@
+class CommentsController < ApplicationController
+  def new
+    @comment = Comment.new
+  end
+
+  def create
+    @comment = Comment.new(comment_params)
+    @comment.author = current_user
+    @comment.post = Post.find(params[:post_id])
+    if @comment.save
+      flash[:notice] = 'Comment added successfully!'
+      redirect_to user_post_path(user_id: params[:user_id], id: params[:post_id])
+    else
+      flash[:alert] = "Couln't add Comment!"
+      render :new, status: :unprocessable_entity
+    end
+  end
```

Comment:
- [ ] Great job for adding the new and create but I can't see any forms in your views to let me be able to create a new comment and I can't see the users even if I am putting the right route in the URL but I am not be able to see any comments it shows me this error could you solve it please? 💯🔥🤩✨

## I created some users and posts and comments with `rails c` but I can't see ⬇⬇

![image](https://user-images.githubusercontent.com/58553711/222934719-6c40f923-5bfe-4497-895c-34e10dde7982.png)

Reply:
this is the correct path http://127.0.0.1:3000/users/1/posts/4
![correct_path](https://user-images.githubusercontent.com/60324033/222935510-1e23f2ee-cf39-4744-a619-3a66374ebf97.PNG)

Reply:
Hi @Zia123456 I am sorry but I showed you what I saw on my local machine as well I sent messages to you to make sure
Could you send it again to review?